setup.py

```diff
@@ -9,15 +9,16 @@
       license='Apache License 2.0',
       packages=find_packages(),
       install_requires=[
+          'numpy>=1.15.4',
```

Comment:
Two files with same list of dependencies is risky. Let's try to make it better.

It would be nice to refer requirements.txt from setup.py like mentioned here https://stackoverflow.com/questions/14399534/reference-requirements-txt-for-the-install-requires-kwarg-in-setuptools-setup-py. 

Warning: you probably will have to also mount setup.py in the same way as docker requirements file in the Dockerfile.